test/torch/mpc/falcon/test_falcon.py

```diff
@@ -23,6 +23,6 @@ def test_conv2d_private(workers):
         bob, alice, james, protocol="falcon"
     )
     assert (
-        falcon.conv2d(x, y, padding=1).reconstruct()
+        falcon.conv2d(x.child, y.child, padding=1).reconstruct()
```

Comment:
Q: Also it conv2d might also need the same type of logic (with wrapper check)

Reply:
working on it

Reply:
done